sopel/trigger.py

```diff
@@ -58,6 +58,9 @@
 ``sender`` property will be ``None``.
 """
 
+DEFAULT_STATUSMSG = ('@',)
```

Comment:
Clients ["SHOULD NOT"](https://modern.ircdocs.horse/#statusmsg-parameter) try to use this feature unless a list of accepted prefixes is advertised, and servers "MUST" specify a value for the 005 token. The default should probably be an empty tuple.

Two older references ([draft-hardy-irc-isupport-00](https://datatracker.ietf.org/doc/html/draft-hardy-irc-isupport-00#section-4.18) and [draft-brocklesby-irc-isupport-01](https://datatracker.ietf.org/doc/html/draft-brocklesby-irc-isupport-01#section-3.11)) reiterate that the 005 token's value is required.

Reply:
Oh, nice catch! I was too focus on that and forgot that there should not be any default. Will fix that.

Reply:
Fixed, with added tests.